qutebrowser/utils/urlutils.py

```diff
@@ -349,6 +337,44 @@ def raise_cmdexc_if_invalid(url):
         raise cmdexc.CommandError(get_errstring(url))
 
 
+def get_path_if_valid(pathstr, cwd=None, relative=False, check_exists=False):
```

Comment:
I've added it thinking that maybe someone could use the method to check if a string looks like a path without checking if actually exists. I thought it might be useful so I agree that is somewhat arbitrary.

If you prefer not to have something like that I can remove the parameter and update the rest of the code accordingly.